lib/screens/v2/candidate_generator/elevator/closures.ex

```diff
@@ -60,14 +60,14 @@ defmodule Screens.V2.CandidateGenerator.Elevator.Closures do
         %Screen{app_params: %ElevatorConfig{elevator_id: elevator_id} = app_params} = config,
         now
       ) do
-    {:ok, alerts} = @alert.fetch_elevator_alerts_with_facilities()
+    {:ok, alerts} = @alert.fetch(activity: "USING_WHEELCHAIR")
```

Comment:
Nit, could use a comment explaining that `"USING_WHEELCHAIR"` filters to only alerts that specifically affect customers who use a wheelchair to board or exit.